src/components/DocumentationTopic.vue

```diff
@@ -172,7 +181,7 @@ import Aside from 'docc-render/components/ContentNode/Aside.vue';
 import BetaLegalText from 'theme/components/DocumentationTopic/BetaLegalText.vue';
 import LanguageSwitcher from 'theme/components/DocumentationTopic/Summary/LanguageSwitcher.vue';
 import ViewMore from 'theme/components/DocumentationTopic/ViewMore.vue';
-import DocumentationHero from 'docc-render/components/DocumentationTopic/DocumentationHero.vue';
+import DocumentationHero from 'docc-render/components/DocumentationTopic/Hero/DocumentationHero.vue';
```

Comment:
We have some other components like `FilterInput` inside the `Filter` folder but I agree that it would be nice to follow a standard for this. Thoughts @mportiz08 ?

Reply:
Any verdict on this?

Reply:
I would leave it as it is. No need of an extra change, in my opinion.

Reply:
I don't have a strong opinion. My weak opinion is that it's slightly easier when we can rely on folder names to map to component names (and its "sub components" within any folder with the same name)—the file structure isn't a blocker to me either way (this pattern does cause issues with the symbol graph script, but I would consider that a bug of the script).